edx_rest_framework_extensions/auth/jwt/cookies.py

```diff
@@ -19,3 +20,17 @@ def jwt_cookie_signature_name():
 
 def jwt_refresh_cookie_name():
     return settings.JWT_AUTH.get('JWT_AUTH_REFRESH_COOKIE') or 'edx-jwt-refresh-cookie'
+
+
+def get_decoded_jwt_from_cookie(request):
```

Comment:
Nit: Rename this to `get_decoded_jwt` for 2 reasons:

1. It's already declared within the `cookies.py` module so it's already scoped to that usage.
2. The input parameter is a `request` and not a `cookie`.

Reply:
done.